examples/sap_hana_ha_simple/main.tf

```diff
@@ -14,17 +14,9 @@
  * limitations under the License.
  */
 
-#
-# Version:    2.0.2022101419281665775728
-# Build Hash: 5f4ef08feb4fed0e1eabc3bfc4b2d64d99001ae7
-#
 module "sap_hana_ha" {
-  # hardcoding the module for testing only
-  source = "../../modules/sap_hana_ha"
-
-  # reference the module by its registry URL and version as such:
-  # source = "terraform-google-modules/sap/google//modules/sap_hana_ha"
-  # version = "0.5.0"
+  source = "terraform-google-modules/sap/google//modules/sap_hana_ha"
```

Comment:
If we switch this in examples, tests will become invalid as they are no longer testing the code changes in the PR rather a static released version. We should use module swapper instead
example PR: https://github.com/GoogleCloudPlatform/terraform-google-secure-cicd/pull/33/files

Reply:
thanks for the tip